Dockerfile

```diff
@@ -57,10 +57,14 @@ RUN sed -i '/disable ghostscript format types/,+6d' /etc/ImageMagick-6/policy.xm
 RUN echo 'export JAVA_HOME=$(readlink -f `which javac` | sed "s:/bin/javac::")/bin/java' >> ~/.bashrc
 
 # Install Node.jx
-ARG NODEJS_VERSION=10
-RUN curl -f --location https://deb.nodesource.com/setup_$NODEJS_VERSION.x | bash - && \
-    apt-get install -yq nodejs && \
-    apt-get install -yq g++ build-essential
+ARG NODEJS_VERSION=10.24.1
+RUN curl -LO https://nodejs.org/dist/v${NODEJS_VERSION}/node-v${NODEJS_VERSION}-linux-x64.tar.xz && \
+  tar -xvf node-v${NODEJS_VERSION}-linux-x64.tar.xz && \
+  cp -r node-v${NODEJS_VERSION}-linux-x64/bin /usr/ && \
```

Comment:
I tried to run: 

```
cp -r node-v${NODEJS_VERSION}-linux-x64/{bin,include,lib,share} /usr/
```
but it is not working, any idea why?

Reply:
This is not macOS related but it depends on the shell executing the command. So it depends on the shell being available in the base docker image.

Reply:
we are using `sh`, but I got the same result with bash 🤔

Reply:
Ah, thanks for the reply, I'll try to confirm wich one we have, but my guess is dash 🤔  (I think I saw something in the console output)

Reply:
I confirm we were using `dash`, but even changing the shell with `SHELL ["/bin/bash", "-c"]` produces the same result. Either way, I'm ok with keeping this a bit more verbose.